setup.py

```diff
@@ -18,7 +18,7 @@
     name="chainlink",  # Required
     # Versions should comply with PEP 440:
     # https://www.python.org/dev/peps/pep-0440/
-    version="0.0.6",  # Required
```

Comment:
I assume this change will require a new release. Maybe not edit this for now and we can update it to 0.0.8 after integrating your change.

Reply:
Oh I see this has 2 separate commits. I will just merge without squashing then.